src/routes/accommodation.route.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import verifyToken from '../middlewares/verify.token.middleware';
+import accommodation from '../controllers/accommodation.controller';
+import Validate from '../helpers/validate.helper';
+import isValid from '../middlewares/validate.middleware';
+
+const router = express.Router();
+
```

Comment:
@VedaDom We also need an endpoint that will take the accommodationID and the userID and returns whether the user has liked or disliked that accommodation. This will help on the frontend.